docs/faqs/NuGet-FAQ.md

```diff
@@ -0,0 +1,387 @@
+- Enter `api.nuget.org` as the hostname and click **Start**.
+- Wait until the **Sent** column is >= 100.
+
+    ![Capturing MTR](media/mtr.png)
+
+- Copy text to clipboard.
+
+*To capture Fiddler:*
+
+- Install the latest version of [Fiddler](http://www.telerik.com/download/fiddler).
+- Start Fiddler and disable capturing traffic using the **File > Capture Traffic** menu.
+- Remove all sessions (select all items in the list, press the **Delete** key).
+- Configure Fiddler to capture HTTPS traffic by checking **Decrypt HTTPS traffic** in the **HTTPS** tab of the **Tools > Fiddler Options...** menu.
+- Close Visual Studio.
+- Enable the **File > Capture Traffic** menu.
+- Start Visual Studio or nuget.exe .exe and perform the actions that are not working. The traffic generated by these actions should show up in Fiddler.
+- Once the actions have run, use **File > Save > All Sessions** to store the captured sessions.
+
+Note: it may be required to set the `HTTP_PROXY` environment variable to `http://127.0.0.1:8888` for routing NuGet traffic through Fiddler.
+
+If that fails, try the [tips mentioned in this StackOverflow post](http://stackoverflow.com/questions/21049908/using-fiddler-to-sniff-visual-studio-2013-requests-proxy-firewall).
+
+**What are the API endpoints for nuget.org?**
+
+- V3: `https://api.nuget.org/v3/index.json`
+- V2: `https://www.nuget.org/api/v2/` (Note that the V2 API is deprecated and does not work with NuGet 4+.)
+
+## NuGet.org Account Management
+
+### How to create a new NuGet account?
+
+To create a NuGet account, you need to have a personal Microsoft account (MSA) or an Azure Active Directory (AAD) account. If you do not have one, you can [create](https://www.microsoft.com/en-us/account) one. Follow the following steps if you have a MSA or AAD account.
+- Go to the [login page](https://www.nuget.org/users/account/LogOn?returnUrl=%2F).
+- Click on `Sign in with Microsoft` button.
+- Enter your MSA/AAD account details. 
+- Please accept the permissions to be given to the `NuGet.org` application.
+- You will be redirected to nuget.org, and asked to register a username.
+- Specify the username in the input box. Please note that the username **is** case sensitive and cannot be changed/renamed later.
+- Click on `Register` button.
+
+You now have a NuGet account. You can perfrom account management on the [account settings](https://www.nuget.org/account) page.
+
+### How to recover NuGet Password login?
+
+Please note that the [NuGet Password login has been discontinued](https://blog.nuget.org/20180515/NuGet.org-will-only-support-MSA-AAD-starting-June.html "NuGet Password login has been discontinued") and the only way to log in to NuGet will be with a personal Microsoft account (MSA) or Azure Active Directory (AAD) account. However, in cases when you are unable to access your associated MSA/AAD accounts you might need to use password logins for recovering your NuGet account. In such situations follow the steps below.
```

Comment:
log in to NuGet -> log in to nuget.org

Reply:
I think we should be using nuget.org in way more places. This refers specifically to our website auth flow, not any other kind of NuGet (e.g. command line) auth.